llvm/Dockerfile

```diff
@@ -44,6 +44,6 @@ RUN cmake -G "Unix Makefiles" \
 RUN make -j"$(nproc --ignore=2)" && make install
 
 
-FROM scratch as final
+FROM alpine as final
```

Comment:
This adds some bulk to the builds, but makes it easier to debug if needing to inspect the image itself